packages/backend/src/controllers/tasks/tasks.util.ts

```diff
@@ -0,0 +1,48 @@
+// Generate random house code
+import { Injectable } from '@nestjs/common';
+import { randomInt } from 'crypto';
+import { TaskDocument } from 'src/db/task/task.schema';
+
+@Injectable()
+export class TaskUtil {
+  selectRandomUser(pool) {
+    return pool[randomInt(0, pool.length)];
+  }
+
+  checkRecurrence(tasks: TaskDocument[]) {
+    const result = [];
+
+    tasks.forEach((task) => {
+      //Need to multiply interval by 1000 to increment the date by seconds.
+      const currentDueDate = new Date(
+        task.dueDate.getTime() + 1000 * task.interval,
+      );
+
+      if (Date.now() > currentDueDate.getTime()) {
+        if (task.assigned == undefined) {
```

Comment:
It this intentionally `==`? Or should it be `=== null` or `=== undefined`?

Reply:
This was to catch more edge cases, if it was `null` or `undefined`